financial-templates-lib/clients/ExpiringMultiPartyClient.js

```diff
@@ -28,11 +28,18 @@ class ExpiringMultiPartyClient {
   getAllPositions = () => this.positions;
 
   // Returns an array of { sponsor, numTokens, amountCollateral } for each position that is undercollateralized
-  // according to the provided `tokenRedemptionValue`.
+  // according to the provided `tokenRedemptionValue`. Note that the `amountCollateral` fed into
+  // `_isUnderCollateralized` is taken as the positions `amountCollateral` minus any `withdrawalRequestAmount`. As a
+  // result this function will return positions that are undercollateralized due to too little collateral or a withdrawal
+  // that, if passed, would make the position undercollateralized.
   getUnderCollateralizedPositions = tokenRedemptionValue => {
-    return this.positions.filter(position =>
-      this._isUnderCollateralized(position.numTokens, position.amountCollateral, tokenRedemptionValue)
-    );
+    return this.positions.filter(position => {
+      const collateralNetWithdrawal = this.web3.utils
+        .toBN(position.amountCollateral)
```

Comment:
nit: going forward, aren't we supposed to be declaring these functions at the top so we can get rid of the `web3.utils`? Note: it's cool with me if you just change the new code and leave other code in the same file using the `web3.utils` syntax.

Reply:
ye I didnt want to make this change in this PR ad we can leave it for one larger chance in issue [1411](https://github.com/UMAprotocol/protocol/issues/1411). Also, this one is a bit more complex because there is no scoped web3 instance so we'll leave it for later.